docs/architecture/dapr-for-net-developers/sample-application.md

```diff
@@ -2,15 +2,15 @@
 title: Introduction to the Traffic Control sample application
 description: An overview of the Traffic Control sample application.
 author: edwinvw
-ms.date: 06/12/2021
+ms.date: 11/17/2021
 ---
 
 # Traffic Control sample application
 
-In the first chapters, you've learned about basic Dapr concepts. You saw how Dapr can help you and your team construct distributed applications while reducing architectural and operational complexity. This chapter introduces the sample application that you'll use to explore the Dapr building blocks.
+In the first chapters, you've learned about basic Dapr concepts. You saw how Dapr can help you and your team construct distributed applications while reducing architectural and operational complexity. This chapter introduces the sample application that you'll use to explore the Dapr building blocks. The application targets .NET 6 and uses the latest C# 10 language features.
 
 > [!NOTE]
-> Download the sample application code from the [Dapr Traffic Control GitHub repo](https://github.com/EdwinVW/dapr-traffic-control). This repository contains a detailed description on how you can run the sample application on your machine.
+> Download the sample application code from the [Dapr Traffic Control GitHub repo](https://github.com/EdwinVW/dapr-traffic-control). This repository contains a detailed description on how you can run it on your machine.
```

Comment:
I actually think the explicit wording is better here than "it" - and likely Acrolinx would flag this use of "it".

Reply:
I will change this.